lib/jekyll-feed/feed.xml

```diff
@@ -74,7 +74,7 @@
         {% unless post_image contains "://" %}
           {% assign post_image = post_image | absolute_url | xml_escape  %}
         {% endunless %}
-        <media:thumbnail xmlns:media="http://search.yahoo.com/mrss/" url="{{ post_image }}" />
+        <media:content xmlns:media="http://search.yahoo.com/mrss/" url="{{ post_image }}" medium="image" />
```

Comment:
What is this about? Can you link to some documentation explaining this?

I don’t want to get rid of `media:thumbnail`, it’s used by feed readers.